tasks/GuildShardDonation/script_task.py

```
@@ -0,0 +1,340 @@
+                    logger.warning("本轮完整扫描未找到目标玩家：%s", self.target_player)
+                    self._return_to_main()
+                    continue
+
+                if not self._click_give_on_same_row(target):
+                    logger.info("目标玩家同行没有‘赠予’按钮，按成功结束")
+                    self._return_to_main()
+                    self._finish(True)
+
+                message = self._wait_and_read_confirm_dialog()
+                if message is None or not self._confirm_message_is_safe(message):
+                    logger.error("确认弹窗安全校验失败，取消捐赠。OCR=%r", message)
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)